src/utils/index.js

```diff
@@ -0,0 +1,11 @@
+export const isExpired = (date) => {
+  if (date === "never" || date === undefined) {
+    return false;
+  }
+  const GivenDate = new Date(date);
+  const CurrentDate = new Date();
+  if (GivenDate > CurrentDate) {
+    return false;
+  }
+  return true;
```

Comment:
I think you could probably turn this into a one-liner.

```suggestion
  return new Date() > new Date(expiryDate)
```